jabsrv/src/main/java/org/jabref/http/JabRefCliStateManager.java

```diff
@@ -0,0 +1,67 @@
+package org.jabref.http;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.collections.ObservableMap;
+
+import org.jabref.logic.search.IndexManager;
+import org.jabref.logic.util.OptionalObjectProperty;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.BibEntry;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JabRefCliStateManager implements CliStateManager {
```

Comment:
Semantically this does not feel right. A Cli cannot have a state. The server may have a state.